src/sqs-consumer.ts

```diff
@@ -102,6 +106,10 @@ export class SqsConsumer {
         this.events.on(event, handler);
     }
 
+    async processMessage(message: Message, options: ProcessingOptions) {
```

Comment:
What is the motivation for having two functions called `processMsg` and `processMessage`, where one just calls the other?

Reply:
one is private, another one is public. It's not a good practice to call public functions from within a private ones. In this particular case it should cause no harm, but may be a bit inconvenient in the future if I just make processMessage public.
It may not be an "official" best practice, but I had to deal with such code in C# in the past, it was a bit of a pain, so from that time I try not to call public methods from private ones.